src/Commands/CloneCommand.php

```diff
@@ -103,14 +105,13 @@ protected function startProgressBar($operation = '')
      * @param \Terminus\Models\Workflow $workflow
      * @param \Symfony\Component\Console\Helper\ProgressBar $progress
      *
-     * @return bool
+     * @return string
      */
     protected function doWorkflow(Workflow $workflow, ProgressBar $progress)
     {
         $workflow->fetch();
         $progress->advance(1);
-        sleep(1);
-        return true;
+        return $workflow->getMessage();
```

Comment:
No sleep till Brooklyn, and also, I like returning something useful...and `true` wasn't that useful...